dhlab-basel/Knora
# doc (webapi): jekyll / paradox based documentation website

### Summary

- [x] add sbt-based documentation generation for:
       - sphinx (old documentation)
       - paradox (new documentation)
       - jekyll (overview website with links to all of documentation)
       - api-v1 typescript
       - api-v2 typescript
       - api-admin swagger
- [x] add sbt-based publishing of documentation to the `gh-pages` branch for http://docs.knora.org
- [x] move `knora.org` site to https://github.com/dhlab-basel/knora.org
- [x] reimplement `knora.org` website using Jekyll (https://github.com/dhlab-basel/knora.org/pull/3)
- [x] add links to `knora.org` for `docs.knora.org` (https://github.com/dhlab-basel/knora.org/issues/2)

### To Test

- inside the `docs` folder run `sbt previewSite`. This should build and open the content for `docs.knora.org` locally.

### Issues

- resolves #733
- resolves #734
